Lesson-4-Task-2/src/test/java/Lesson4/Task2/CircleTest.java

```diff
@@ -0,0 +1,26 @@
+package Lesson4.Task2;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+public class CircleTest {
+    @Test
+    public void getAreaCircleTest(){
+    //given
+    int r =2;
+    // when
+    Circle circle = new Circle(r);
+    double result = circle.getArea();
+    //then
+        Assertions.assertEquals(12.57,result);
+    }
+    @Test
+    public void getPerimeterCircleTest(){
+        //given
+        int r =2;
+        // when
+        Circle circle = new Circle(r);
```

Comment:
wywołanie konstruktora to nie coś co tutaj testujemy, więc lepiej będzie jak damy to w //given

Reply:
Poprawione